doc/source/license.rst

```diff
@@ -1,40 +1,39 @@
-License
-=======
+:tocdepth: -1
+
+.. index:: license
 
-This program is part of the DiffPy open-source project at Columbia
-University and is available subject to the conditions and terms laid out 
-below.
+License
+#######
 
-Copyright © 2009-2019, Trustees of Columbia University in the City of 
-New York, all rights reserved.
+OPEN SOURCE LICENSE AGREEMENT
+=============================
+BSD 3-Clause License
 
-For more information please visit the diffpy web-page at 
-http://diffpy.org or email Prof. Simon Billinge at sb2896@columbia.edu.
+Copyright (c) {% now 'utc', '%Y' %}, The Trustees of Columbia University in 
```

Comment:
add earlier  begin date and make sure it is consistent between the readme and here.

Reply:
<img width="748" alt="Screenshot 2024-08-10 at 7 16 54 PM" src="https://github.com/user-attachments/assets/fd0f9d70-2500-4ccf-8cdf-d8b43eb5e7c3">

Added 2009 (earlier date) to where we have copyright